tests/fees/test_fees.py

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
Please ignore this file for now. The `test_fees` test is skipped and will be completed in #754